package.json

```diff
@@ -131,6 +131,7 @@
     "@types/eventsource": "1.0.1",
     "@types/express": "4.11.1",
     "@types/jest": "22.2.3",
+    "@types/node": "10.1.4",
```

Comment:
Could you point me where do we use it? 👀

Reply:
It's used when ever we work with the native node API... 

-> https://github.com/mesosphere/mockserver/pull/77/files#diff-42fc62e35f75bbed1464cc0172a219e2R1
-> https://github.com/mesosphere/mockserver/pull/77/files#diff-916156c481141bfcd08c6ae40ce10e00R65
-> https://github.com/mesosphere/mockserver/pull/77/files#diff-cf5710c14170fce08e627b8c67aec49bR1